tests/functionaltests/FunctionalTests.cpp

```diff
@@ -278,4 +280,27 @@ class NSBundle {
     TEST_METHOD(NSBundle_MSAppxURL) {
         NSBundleMSAppxURL();
     }
+}; /* class NSBundle */
+
+//
+// Cortana Activation Tests
+//
+
+extern void CortanaTestVoiceCommandForegroundActivation();
+
+class Cortana {
+public:
+    BEGIN_TEST_CLASS(Cortana)
+    TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
+    TEST_CLASS_PROPERTY(L"UAP:Host", L"Xaml")
+    TEST_CLASS_PROPERTY(L"Ignore", L"true")
```

Comment:
The test doesn't test anything yet, so I've disabled it for now

---
In reply to: [68832198](https://github.com/Microsoft/WinObjC/pull/592#discussion_r68832198) [](ancestors = 68832198)